### PR TITLE
Updates to IGCE schema

### PR DESCRIPTION
**Pull request by jasonburkert-ccpo**
Small updates to IGCE schema added in #1059

- Changes types of properties `surge_capabilities` and `months_in_period`
- Includes expected ranges in descriptions of properties `surge_capabilities` and `months_in_period` without limiting accepted input
- Fixes typo in description of schema object `IndependentGovernmentCostEstimate`
- Updates property descriptions for clarity
- Includes some property examples for clarity

Ticket: AT-7757

### CODE REVIEW

File: hoth_api.yaml [hunk 1]
```diff
@@ -445,39 +447,45 @@ components:
       type: object
       properties:
         period:
-          description: "The period of performance for a specific period sheet."
+          description: "A base or option period."
```
Comment (juliusfitzhugh-ccpo) [inline hoth_api.yaml:450]: Not sure if we want to remove the `" "` for the `description` like you did for the other ones.

**Review by juliusfitzhugh-ccpo — APPROVED** [review]
Thanks for providing clearer descriptions and examples! I was not sure how much to add or not add, but I like the changes you've made. Also, good catch on the `integer` update.

File: hoth_api.yaml (hunk 1 above)
Comment (jasonburkert-ccpo) [reply hoth_api.yaml:450]: We may need to do that for the entire file someday.  There's definitely a mix.

**Comment by jasonburkert-ccpo** [comment]
> Thanks for providing clearer descriptions and examples! I was not sure how much to add or not add, but I like the changes you've made. Also, good catch on the `integer` update.

I have a tendency to add too much business information.  But a little is fine.

**Comment by jasonburkert-ccpo** [comment]
FYI this is hours old, forcing the update to `.secrets.baseline`
https://github.com/Yelp/detect-secrets/releases/tag/v1.3.0

**Review by jeffsegal-ccpo — APPROVED** [review]

File: .secrets.baseline [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (juliusfitzhugh-ccpo) [inline .secrets.baseline:2]: Seems like the `.pre-commit-config.yaml` needs to be updated as well to `v1.3.0` to run it correctly locally.

**Review by juliusfitzhugh-ccpo — CHANGES_REQUESTED** [review]
The change is geared towards fixing the `.secrets.baseline` issue.

File: .secrets.baseline (hunk 2 above)
Comment (jasonburkert-ccpo) [reply .secrets.baseline:2]: Updated in a333f0215663c042f70fea26a774362968a8faa2

**Review by juliusfitzhugh-ccpo — APPROVED** [review]

**Pull request merged**